### DISCUSSION TITLE
Modified scoring rules

### DISCUSSION THREAD

**Discussion by s-zanella** [ethz-spylab/satml-llms-ctf-issues#54] [Questions]
The latest scoring rules attempt to eliminate the rush towards being among the first teams to break a defense. However, they only affect the bonus, not the score multiplier. I believe that the exponential decay of $v_D$ vastly outweighs this.

The current choice of γ=0.85 heavily penalizes late entrants: it makes it almost worthless to be, say, the 20th team to break a defense (even during the first 36h). Perhaps setting γ=1 (no exponential decay) or very close to 1 would be fairer? That way, a team that isn't as fast would have a chance of overtaking others as long as they break overall more defenses over the course of the whole attack phase. Otherwise, a team breaking all 44 defenses could end up losing out against a team that breaks just 22 but does it before any others.

**Comment by regoar** [comment]
Maybe I missed something but I think $v_D$ is the same for all teams when calculating the final score, no matter if you solved first or not. The first team will have only an additional $b_D \cdot v_d$. From the rules:
> Note that vD is continuously updated, hence the score for any (team, defense) pair changes every time another team breaks that defense.

**Comment by dedeswim** [comment]
@regoar is correct. Once you break a defense, your score is not fixed as it is dynamically updated as $v_D$ changes (i.e., as other teams break the same defense).

Let's say you break a defense as the first team, with just one chat, then $v_D = 0.85$, and you **provisionally** get $(1000+200) \cdot 0.85 = 1020$ points. If the defense is broken by someone else (let's assume also with just one chat and after 36 hours to simplify the computations), then the defense value goes down to $v_D = 0.85 \cdot 0.85 = 0.7225$, which means that the other team gets $(1000+100) \cdot 0.7225 \simeq 795$ and your points are decreased to $(1000+200) \cdot 0.7225 \simeq 867$. **This score is still provisional**, subject to other teams breaking the same defense.

**Comment by s-zanella** [comment]
I had misread the rules, thanks both for the clarificationI 
I've no concern about the exponential decay of $v_D$ then and the choice of $\gamma=0.85$ is sensible.
